### PR TITLE
Add arrow function returning object.

### PR DESCRIPTION
**Pull request by oskarcieslik**
Imo this looks better, than something like:

```javascript
[1, 2, 3].map((number, index) => {
  return {
    index: number
  }
});
```

What do you guys think? :) If you think that's bad style, maybe you should add this example with `//bad` comment?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -826,6 +824,11 @@ Other Style Guides
       const nextNumber = number + 1;
       return `A string containing the ${nextNumber}.`;
     });
+
```
Comment (ljharb) [inline README.md]: please remove this trailing whitespace

**Comment by ljharb** [comment]
Since our linter rules now prevent most developer errors in this scenario, I think this is OK.

Could you rebase it on top of the latest master?

**Comment by oskarcieslik** [comment]
K, removed trailing whitespace & updated with master :)

**Comment by ljharb** [comment]
@oskarcieslik thanks! this PR does contain a merge commit tho - could you rebase it on top of latest master, and force push, instead?

**Pull request closed without merging**